neighbour-model/src/main/java/no/vegvesen/ixn/federation/model/Neighbour.java

```diff
@@ -47,8 +47,12 @@
 	private SubscriptionRequest fedIn = new SubscriptionRequest(SubscriptionRequestStatus.EMPTY, new HashSet<>());
 
 	@OneToOne(cascade = CascadeType.ALL, fetch = FetchType.EAGER, orphanRemoval = true)
-	@JoinColumn(name = "con_back", referencedColumnName = "id", foreignKey = @ForeignKey(name = "fk_neighbour_connection_backoff"))
-	private ConnectionBackoff connectionBackoff = new ConnectionBackoff();
+	@JoinColumn(name = "mes_back", referencedColumnName = "id", foreignKey = @ForeignKey(name = "fk_message_connection_backoff"))
+	private ConnectionBackoff messageConnectionBackoff = new ConnectionBackoff();
```

Comment:
We have discussed a new name for ConnectionBackoff: Connection!
Backing off is something the Connection does internally.